src/libraries/Common/src/System/Security/Cryptography/Asn1/asn.xslt

```diff
@@ -51,6 +51,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Why is this necessary? Are these files included in a project that's not yet annotated?

Reply:
It may not be; one of the other XSLT templates in the same file had `#nullable enable`, so some of the files already had them:

https://github.com/dotnet/runtime/blob/419e949d258ecee4c40a460fb09c66d974229623/src/libraries/Common/src/System/Security/Cryptography/Asn1/asn.xslt#L130

Would it be preferable to get rid of all of them if possible now?

Reply:
If they're no longer necessary, yes, please.

Reply:
There are 5 projects that have `AsnXml` in them and they are have `<nullable>enable</nullable>`, so it appears that `#nullable enable` does not need to be in the template anymore.

Reply:
Cool.